.github/workflows/frontend.yml

```diff
@@ -78,7 +75,6 @@ jobs:
       name: Run ESLint linter
 
   fe-linter-docs-links:
-    if: "!contains(github.event.head_commit.message, '[ci skip]')"
```

Comment:
Maybe too much nitpicking/optimization, but we can probably avoid checking dock links if path is not `docs/**`?

Reply:
Unfortunately path conditional doesn't work well at the job level (compared to the workflow level).

If we really really need this (saves 3 minutes  perhaps), we can at some point spawn `fe-linter-docs-links` to its own separate workflow (another YAML file).

Reply:
We can definitely live without this.
Like I said, it was probably pushing it too far, anyway.

What you did here is HUGE improvement already. 💪